File: 강재연/programmers/P181900.js
// 문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181900
// 시간: 0.05 ms
// 메모리: 33.4 MB

function solution(my_string, indices) {
    const answer = [...my_string];
    indices.forEach((str) => {
        answer[str] = '';
    });
    
    return answer.join('');
}

File: 공예진/programmers/P181900.java
//문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181900
//시간: 0.52ms
//메모리: 87.8MB

import java.util.*;

class Solution {
    public String solution(String my_string, int[] indices) {
        String answer = "";
        StringBuilder sb = new StringBuilder(my_string);

        Arrays.sort(indices);
        for (int i = indices.length - 1; i >= 0; i--) {
            sb.deleteCharAt(indices[i]);
        }

        return sb.toString();
    }
}